CONTRIBUTING.md

```diff
@@ -25,7 +27,8 @@ Our style rules and PR reviews are based loosely on Autosar's `Guidelines for th
 * Doxygen should compile with no warnings when run at the root of the project with the command `doxygen doxyfile`
 * Absolutely no code shall be added that is under a more strict license than MIT or which has not had conditions met to be distributed under our license
 * Builds must pass the compilation github action
-* C++14 or earlier is required to help provide the maximum compatibility to all "modern" compilers
+* C++14 or newer is required to help provide the maximum compatibility to all "modern" compilers
```

Comment:
Oh, one thing, when I specified `earlier` I did mean `older` not newer. The idea was to maintain compatibility with embedded compilers, many of which support 14 at the newest. For example ARM Compiler 6

Reply:
<img width="719" alt="image" src="https://user-images.githubusercontent.com/10929341/220449479-2fd0ff42-da23-49ec-8b39-eae2e22e8208.png">

Reply:
Ohh that totally makes sense! I'll change it back

Reply:
Maybe it's more clear to state?:
```suggestion
* Only features present in C++14 can be used to help provide the maximum compatibility to all "modern" compilers
```

Reply:
That also works!